learning-bitmark/working-with-bitmarks/issuing-bitmarks.md

```diff
@@ -0,0 +1,134 @@
+# Registering properties on the Bitmark blockchain
+
+The following records are required to register a property on the Bitmark blockchain
+
+* *Asset Record* - contructed by the Asset Registration process
+
+* *Issue Records* - contructed by the Bitmark Issuance process.
+
+<br>
+<br>
+## Registering properties using Bitmark App
+
+* Download and install the [Android](https://apps.apple.com/us/app/bitmark-property-registry/id1429427796) or [iOS](https://apps.apple.com/us/app/bitmark-property-registry/id1429427796) Bitmark App
+
+* Create a Bitmark Account
+
+* Go to Properties screen
+
+* Tap on the **+** icon
+
+* Select a file or photo as the expected asset
+
+* Fill in the required fields
+
+* Tap **ISSUE** button
+
+As tapping on the **ISSUE** button, the Bitmark app submits the requests of asset registration and bitmark issuance to the Bitmark blockchain. It will takes several minutes for the bitmarks to be confirmed on the blockchain.
+
```

Comment:
This section makes it even more obvious to me that it would be a big benefit to have the screen shots, though I know they're time-consuming to put together.

Reply:
Added screen shots.